src/main/resources/entity-types/finance/drv_ledger_details.json5

```diff
@@ -0,0 +1,340 @@
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'restrictEncumbrance'"
+    },
+    {
+      name: 'restrict_expenditures',
+      dataType: {
+        dataType: 'booleanType',
+      },
+      values: [
+        {
+          label: 'True',
+          value: 'true',
+        },
+        {
+          label: 'False',
+          value: 'false',
+        },
+      ],
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'restrictExpenditures'"
+    },
+    {
+      name: 'created_date',
+      dataType: {
+        dataType: 'dateType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'createdDate'"
+    },
+    {
+      name: 'updated_date',
+      dataType: {
+        dataType: 'dateType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'updatedDate'"
+    },
+    {
+      name: 'created_by_user_id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'createdByUserId'"
+    },
+
+    {
+      name: 'updated_by_user_id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'updatedByUserId'"
+    },
+    {
+      name: 'allocated',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'allocated'"
+    },
+    {
+      name: 'available',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'available'"
+    },
+    {
+      name: 'net_transfers',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'netTransfers'"
+    },
+    {
+      name: 'created_by_username',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'createdByUsername'"
+    },
+    {
+      name: 'updated_by_username',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'updatedByUsername'"
+    },
+    {
+      name: 'unavailable',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'unavailable'"
+    },
+    {
+      name: 'initial_allocation',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'initialAllocation'"
```

Comment:
Same as above. This field is a numberType so we need to return it as a float instead of as text.